### ISSUE TITLE
[Bug] Intelligently hidden dock revealed by a captured cursor within a windowed game

### ISSUE THREAD

**Issue by Kartonrealista** [pop-os/cosmic-dock#50]
**Version** - Pop!_OS 21.04 x86_64

**Issue:**
The dock reveals itself when a mouse is moved in its direction (on windowed mode) in a game that captures the cursor (here Minecraft).
![Zrzut ekranu z 2021-08-03 19-31-00](https://user-images.githubusercontent.com/63356574/128060396-87aa80d9-1aa0-412c-8146-55cdebcd5dd5.png)

**Expected behaviour:**
 It should not reveal the dock, since you're not moving a cursor across the screen in any visible way.

**Comment by noobishply** [comment]
This happened to me in 20.10 with Dash to Dock, and I was able to fix it by enabling (or disabling, forgot) raw input inside Minecraft by going to Options > Controls > Mouse Settings > Raw Input
I have just tried this and it seems to work (turning off Raw Input), but if you push your mouse towards the dock really hard/fast, then it seems to pop up for a second. Definitely won't be accidentally triggering it, though. 
Hope this helped!
